core/src/main/java/org/jsmart/zerocode/core/utils/HelperJsonUtils.java

```diff
@@ -93,13 +93,6 @@ public static String createAndReturnAssertionResultJson(int httpResponseCode,
         return formattedStr;
     }
 
-    private void setRequestHeaders(Object headers, ClientRequest clientExecutor) {
-        Map<String, Object> headersMap = HelperJsonUtils.readObjectAsMap(headers);
-        for (Object key : headersMap.keySet()) {
-            clientExecutor.header((String) key, headersMap.get(key));
-        }
-    }
-
```

Comment:
Was this an unused method? Is that why we are removing it?

Reply:
@baulea , suggest not to remove this, if it is not blocking you. Not sure, but this may be used by some users/devs etc.

Reply:
How can the removal of an unused private method do any harm? It is not visible from outside, even in child classes.

I removed this method because the parameter type org.jboss.resteasy.client.ClientRequest has been removed from the upgraded resteasy version (is replaced by javax.ws.rs.client.Client).

Reply:
Looks good to me.